quick_client/test/test_application_creation_data.py

```diff
@@ -0,0 +1,65 @@
+# coding: utf-8
```

Comment:
It looks like we have disabled the generation of tests so far, see https://bakdata.github.io/quick/dev/developer/development/#openapi

Reply:
Sry my bad I used actually the docs but for some reason the last line of the command was not applied in my terminal 😄